### ISSUE TITLE
Syncing fails - no real time information

### ISSUE THREAD

**Issue by Yky** [MihaiBalint/TimisoaraPublicTransport#6]
The app fails to sync with either of the two available URLs in the settings. This means the main functionality of the app is not working, as there is no real-time information.

People on the Play Store are complaining about the same issue:
https://play.google.com/store/apps/details?id=ro.mihai.tpt&hl=en

**Comment by MihaiBalint** [comment]
The app source code has moved to a private repo where it’s being
refactored. The google play version uses code from that repo that’s why it
sometimes works.

We are working with the transport agency to resolve the reliability issue
and build an open API that provides the relevant data:
https://opentsr.com/

If you would like to receive updates about the status of the API please
fill this form:
https://forms.gle/1CVQXBY6Gz3wLegz6

**Comment by Yky** [comment]
Thank you for your quick response.

There is an F-Droid issue about this app here: https://gitlab.com/fdroid/fdroiddata/issues/1731

Are you still planning on providing an open-source version of the app for F-Droid?

**Comment by MihaiBalint** [comment]
Yes I want to provide an open source version. I’m also comfortable
resubmitting the app to F-Droid rather than keeping the issue open. What is
your suggestion?

**Comment by IzzySoft** [comment]
F-Droid maintainer here. If you say the app we currently have is "broken" (i.e. cannot work, for whatever reason), we could (temporarily) move it out of the main repo and into our archive (that's what we normally do with apps no longer working). Once you have a new version ready for us, that move can easily be reverted (it's nothing but a marker in the metadata file saying in which part the APKs should show up, put to simple terms), and the new version added. That way we avoid "disappointed users".

Apart from that, wouldn't it be a good idea to keep the issue(s) open as a reminder?

**Comment by MihaiBalint** [comment]
It’s settled then, please archive the app for now but keeping the issue
open as you suggested.

Thanks for all your work.

**Comment by IzzySoft** [comment]
Thanks Mihai, marker set (will do the necessary steps when I'm back at my machine; should take effect within a week – that is, when a new index is being built and deployed). Please ping me here again when your FOSS flavor is ready (or if you have any questions in between, of course).

**Comment by IzzySoft** [comment]
[Done](https://gitlab.com/fdroid/fdroiddata/merge_requests/5208). Will take effect with the next index build. Looking forward then to your "ping" for re-activation :wink: Thanks again!
